lib/ecto/pools/sojourn_broker/worker.ex

```diff
@@ -41,6 +41,9 @@ defmodule Ecto.Pools.SojournBroker.Worker do
     backoff_threshold = div(max_backoff, 3)
     lazy = Keyword.get(worker_opts, :lazy, true)
 
+    # Seed random number generator for backoff
+    :random.seed(:erlang.now)
```

Comment:
Are we relying on the different workers having different seeds? If not, `:os.timestamp` is fine. If you need something else, maybe even the last example with phash2 of `self()`?